workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/1000-setting-up.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "パイプライン入門"
+weight = 110
++++
+
+> 注意 : このワークショップの章は、前の全ての章を完了していることを前提としています。完了していないか、またはこの部分のみ検証する場合は、[こちら](https://github.com/aws-samples/aws-cdk-intro-workshop/tree/master/code/python/main-workshop) のコードを利用すれば、テストを追加した直後のプロジェクトの状態に設定することができます。
+
+## パイプラインのスタックの作成
+最初のステップでは、パイプラインを持つスタックを作成します。
+実際の「本番」アプリケーションとは分離するので、完全に独立させる必要があります。
+
+`cdk_workshop` フォルダの中で `pipeline_stack.py` という新しいファイルを作成します。ファイルに以下の内容を追加します。
+
+{{<highlight python>}}
+from constructs import Construct
+from aws_cdk import (
+    Stack
+)
+from pipeline_stage import WorkshopPipelineStage
```

Comment:
I don't think the line `from pipeline_stage import WorkshopPipelineStage` is supposed to be here quite yet 🙂